src/cap32.cpp

```diff
@@ -2151,25 +2121,9 @@ int cap32_main (int argc, char **argv)
               dword cpc_key[2] = {0xff, 0xff};
               bool release = false;
               CPC.InputMapper->CPCkeyFromJoystickAxis(event.jaxis, cpc_key, release);
-			  if (!CPC.paused && cpc_key[0] != 0xff) {
-                if(release) {
-                 keyboard_matrix[static_cast<byte>(cpc_key[0]) >> 4] |= bit_values[static_cast<byte>(cpc_key[0]) & 7]; // key has been released
+              applyKeypress(cpc_key[0], keyboard_matrix, !release);
+              if (!CPC.paused && cpc_key[0] != 0xff && release) {
                  keyboard_matrix[static_cast<byte>(cpc_key[1]) >> 4] |= bit_values[static_cast<byte>(cpc_key[1]) & 7]; // key has been released
```

Comment:
Do we need to keep this line ?
Shouldn't it be something like:
if (!CPC.paused) {
  if (cpc_key[0] != 0xff) {
    applyKeypress(cpc_key[0], ...)
  } else {
    applyKeypress(cpc_key[1], ...)
  }
}

In which case, should we also check cpc_key[1] != 0xff ?